openai_text_utils.py

```diff
@@ -1,17 +1,39 @@
 import api_gen_properties
 import openai
+import card_arguments
 
 openai.api_key = api_gen_properties.API_KEY
 
-def generate(prompt):
-    request = build_request(prompt)
+prompt = """Create a new randomly generated Magic the Gathering card with the name """
+json_format = """\nCreate a valid json object for the card into the following format:
+{
+    "Name": "Card Name",
+    "Rarity": "Rarity of Card",
+    "Type": "The card Types in an array",
+    "Manacost": "The manacost of the card with colors",
+    "CMC": The converted manacost of the card,
+    "Supertype": "The supertype of the card",
+    "Subtype": "The subtypes of the card in an array",
+    "Colors": "The colors of the card in an array",
+    "Power": The power of the card,
+    "Toughness": The toughness of the card,
```

Comment:
Numeric fields should be in quotes too.

They're numbers, but we want to treat them as Strings because we're not doing any operations on them